src/modules/render/comments.js

```diff
@@ -0,0 +1,132 @@
+import API from '../data';
+import counter from '../counter';
+
+const modal = document.getElementById('modal-comment');
+
+const displayComments = (ul, userComment) => {
+  const comment = document.createElement('li');
+  comment.className = 'comment-li';
+  const commentMeta = document.createElement('div');
+  commentMeta.className = 'comment-meta';
+  comment.appendChild(commentMeta);
+  const name = document.createElement('p');
+  name.style.textTransform = 'Capitalize';
+  name.innerText = userComment.username;
+
+  const date = document.createElement('p');
+  date.className = 'no-comments';
+  date.innerText = userComment.creation_date;
+  commentMeta.appendChild(name);
+  commentMeta.appendChild(date);
+  const message = document.createElement('p');
+  message.innerText = userComment.comment;
+  message.style.textTransform = 'Capitalize';
+  comment.appendChild(message);
+  ul.appendChild(comment);
+};
+
+const commentModal = (meal) => {
```

Comment:
- [ ] There is some text in the right that seems to overlap the comments modal. I noticed that in the video it was ok but when it came to my screen this is what I can see. Is it because I am on a smaller screen? If so then incase this becomes an issue to another reviewer I guess you can improve it by making the text go below the image as you did in the reservations popup. Let me know what you think.
![image](https://user-images.githubusercontent.com/68167810/133826898-f4eaecf5-ca10-4895-8e8d-b211224e0496.png)